.github/workflows/lint.yml

```diff
@@ -38,3 +38,25 @@ jobs:
 
       - name: Run Rubocop
         run: bin/bundle exec rubocop
+
+  terraform:
+    name: Terraform
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 1.0.10
+
+      - name: Check formatting
+        run: terraform fmt -check
+        working-directory: terraform
+
+      - name: Validate
+        run: |
+          terraform init -backend=false
+          terraform validate -no-color
+        working-directory: terraform
```

Comment:
What does this do exactly @merrilmathew and should we keep it in the linting script? Should we have it in the deploy script as well?

Reply:
This simply validates the terraform files and configuration